cpp/Platform.Timestamps/CommonEraClock.h

```diff
@@ -13,45 +13,45 @@ namespace Platform::Timestamps
         using period = duration::period;
         using time_point = std::chrono::time_point<common_era_clock>;
         static constexpr bool is_steady = false;
-        static constexpr std::uint64_t ticks_after_anno_domini = 621355968000000000; // the number of ticks passed from 0001/01/01 12:00 to 1970/01/01 00:00
+        static constexpr std::uint64_t ticks_between_anno_domini_and_unix_epoch = 621355968000000000; // the number of ticks passed from 0001/01/01 12:00 to 1970/01/01 00:00
 
         static duration time_since_epoch()
         {
             return std::chrono::duration_cast<duration>
-                (std::chrono::system_clock::now().time_since_epoch()) + duration(ticks_after_anno_domini);
+                (std::chrono::system_clock::now().time_since_epoch()) + duration(ticks_between_anno_domini_and_unix_epoch);
         }
 
         static time_point now()
         {
             return time_point(time_since_epoch());
         }
 
-        static std::chrono::sys_seconds to_sys(const time_point &common_time_point)
+        static std::chrono::system_clock::time_point to_sys(const time_point& time_point)
```

Comment:
It looks like this syntax: 
```cpp
using int_alias = int;
int_alias int_alias = 12;
```
If you wants to use this then rewrite to:
```cpp
... to_sys(const common_era_clock::time_point& time_point)
```

Reply:
I agree.